docs/proposals/proposal-workflow-authentication.md

```diff
@@ -0,0 +1,69 @@
+# Proposal for Workflow Authentication
+
+## Proposal summary
+
+**For workflow-to-workflow-service authentication, use:**
```

Comment:
As per my other comment, the terminology needs adjusting? ("workflow-to-workflow-service"??)

Reply:
Sure :grimacing: I suspect I have misunderstood some important aspect here as I am not sure how the UI Server comes into this, I suspect I should not have included the UI as part of the user client scope below & then the topic would just boil down to the CLI for both job & user clients.

I'll look at the comments below & get further clarification from @matthewrmshin today. Sorry for the confusion!

Reply:
No, I initially thought you meant "WFS to UIS" but that was before I read your document.

I think you actually mean "CLI to WFS" (for CLI commands invoked by jobs or the suite owner, direct to WFS without going via the UIS).

The confusing bit in your description is "**workflow** to workflow service" - what is "workflow" (in bold) here?  I think you mean "job or owner CLI".